docs/mkdocs.yml

```diff
@@ -92,298 +91,229 @@ plugins:
                 - "!^_"
 
 nav:
-  - LangGraph:
-    - index.md
+  - Guides:
+    - Home: index.md
     - Get started:
-      - Learn the basics: tutorials/introduction.ipynb
+      # - TO DO: Quickstart with Prebuilt Agents
+      - LangGraph basics: tutorials/introduction.ipynb
+      - Key features: concepts/high_level.md
```

Comment:
we'd need the titles to be consistent - either rename this to "Why LangGraph" or rename "high_level.md" to "Key Features"

Reply:
Ended up removing this page and combining its content into the README/index